src/main/webapp/app/overview/visualizations/exercise-scores-chart/exercise-scores-chart.component.ts

```diff
@@ -154,30 +154,25 @@ export class ExerciseScoresChartComponent implements AfterViewInit, OnChanges {
      * @param data the event sent by the framework
      */
     onSelect(data: any): void {
```

Comment:
Can we somehow replace this `any` with an actual type?

Reply:
Fixed this in the most recent commit 👍